lib/matplotlib/backends/backend_ps.py

```diff
@@ -1440,27 +1440,32 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     command = '%s cd "%s" && latex -interaction=nonstopmode "%s" > "%s"'\
```

Comment:
I didn't want to try it as I couldn't test the `precmd` part on Linux.
Is that taken care of automatically with using `cmd` or would you still need the `precmd` somewhere else still?